app/views/secondary_content_links/index.html.erb

```diff
@@ -40,48 +40,50 @@
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-full">
+            <%= render "shared/steps/form_errors", resource: @step_by_step_page %>
 
-          <%= render "govuk_publishing_components/components/heading", {
-            text: "Add new secondary link",
-            margin_bottom: 5
-          } %>
+            <%= render "govuk_publishing_components/components/heading", {
+              text: "Add new secondary link",
+              margin_bottom: 5
+            } %>
 
-          <%= render "govuk_publishing_components/components/input", {
-            label: {
-              text: "Base Path"
-            },
-            hint: capture do %>
-              For example: 
-              <a href="https://www.gov.uk/pay-vat" target="_blank" class="govuk-link">https://www.gov.uk/pay-vat</a> 
-              or <a href="https://www.gov.uk/bank-holidays" class="govuk-link" target="_blank">/bank-holidays</a>
-            <% end,
-            name: "base_path"
-          } %>
+            <%= render "govuk_publishing_components/components/input", {
+              label: {
+                text: "Base Path"
+              },
+              hint: capture do %>
+                For example:
+                <a href="https://www.gov.uk/pay-vat" target="_blank" class="govuk-link">https://www.gov.uk/pay-vat</a>
+                or <a href="https://www.gov.uk/bank-holidays" class="govuk-link" target="_blank">/bank-holidays</a>
+              <% end,
+              name: "base_path"
+            } %>
+          </div>
         </div>
-      </div>
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-          <%= render "govuk_publishing_components/components/button", {
-            text: "Add secondary link"
-          } %>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-full">
+            <%= render "govuk_publishing_components/components/button", {
+              text: "Add secondary link"
+            } %>
+          </div>
         </div>
-      </div>
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full govuk-!-margin-top-3">
-          <%= link_to 'Cancel', step_by_step_page_secondary_content_links_path, class: "govuk-link add-secondary-link-cancel--link" %>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-full govuk-!-margin-top-3">
+            <%= link_to 'Cancel', step_by_step_page_secondary_content_links_path, class: "govuk-link add-secondary-link-cancel--link" %>
+          </div>
         </div>
-      </div>
-    <% end %>
-  </div>
+      <% end %>
+    </div>
+  <% end %>
 </div>
-
-<hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
-
+<% if !@step_by_step_page.scheduled_for_publishing? %>
+  <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+<% end %>
```

Comment:
Can the conditional above this be expanded to include the section break?

Reply:
This wouldn't work because there is an if-else statement below, which determines whether Schedule or Unschedule button is shown, so the 'Unschedule publishing' button would never appear.